src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const Books = () => {
+  const booksArray = [{
+    id: 1, title: 'Dune', author: 'Frank Herbert', genre: 'Science Fiction',
+  }];
+  return (
+    <>
+      <Navbar />
+      <h1 className="books-page-title">E-Bookopolis</h1>
+      <form id="form">
+        <input type="text" id="book-title" name="book-title" className="book" placeholder="Book Name..." />
+        <select name="categories" id="categories-dropdown">
+          <option value="" disabled selected hidden>Categories</option>
+          <option value="horror">Horror</option>
+          <option value="sci-fic">Sci-Fic</option>
+          <option value="comedy">Comedy</option>
+        </select>
+        <input type="submit" value="Submit" />
+      </form>
```

Comment:
- [ ] Let's refactor your **Books.js** file to look neater. Kindly extract this form into a new component and render that component here instead.

Reply:
I have implemented the change.